### ISSUE TITLE
Failed to load resource http://localhost:8081/shoe/info

### ISSUE THREAD

**Issue by ralt** [Raynos/browserify-server#4]
Hi,

I think I got it working. There is an index.js file in cwd, and it does watch changes and reloads.

However, the live-reload doesn't work, as I'm getting the error in the title in the browser's console.

I'm using [this index.js file](https://github.com/Ralt/shooter/blob/844a208caa848e761019a171ca56719bff4c4fe5/index.js).

**Comment by Raynos** [comment]
@Ralt that should work if your browserify-server is running.

**Comment by ralt** [comment]
@Raynos Well it fails with this code: https://github.com/Ralt/shooter/tree/844a208caa848e761019a171ca56719bff4c4fe5

When I use the following command:

```
browserify-server --folder=.
```

It serves the files correctly, but it spits out this error in the browser's console.

**Comment by Raynos** [comment]
@Ralt  [works for me](https://github.com/Ralt/shooter/pull/1)

**Comment by ralt** [comment]
Doesn't for me :| I added the same structure as in your commit (https://github.com/Ralt/shooter), but there is the same console error as in http://raynos.github.com/shooter.

Even weirder, I deleted the node_modules folder and ran `npm install` to have everything fresh, and now the watching throws an error on the _second change_. The first change saves and creates the new bundle, the second throws an error about no such file or directory: `js/map.js~`. Or it doesn't throw an error, but it stops watching changes.

It looks like I'm missing something big. I guess something is wrong with my pc, but I can't see what. I'm using Ubuntu with node 0.8.7.
